practice-two/src/styles/pages/home/places.css

```diff
@@ -0,0 +1,112 @@
+/*------------------------------------*\
+  #PLACES SECTION
+\*------------------------------------*/
+
+.places {
+  padding-top: calc(37 / 16 * 1rem);
+  padding-bottom: 2.5rem;
+}
+
+.places .container {
```

Comment:
Why do we need style for container?

Reply:
Because the container is nested in the class places so when we want to style layout for Places Section, we have to style for container

![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/a2e5fc09-7baf-48e5-bc1e-f7267d3f8944)